src/utils/watch/fastify.ts

```diff
@@ -0,0 +1,55 @@
+import C from 'ansi-colors'
```

Comment:
colorette was 3 times faster, last time I checked, and we use it in pino-pretty already.

Reply:
I would avoid any tools using `"type": "module"`. Which ease the problem of `dependabot` always bumping those package but will never be success.

Reply:
@climba03003 the main entry point of colorrette is a cjs file: https://github.com/jorgebucaran/colorette/blob/main/package.json#L5.